localstack/services/cloudformation/models/cloudformation.py

```diff
@@ -15,32 +15,12 @@ class CloudFormationStack(GenericBaseModel):
     def cloudformation_type():
         return "AWS::CloudFormation::Stack"
 
-    def fetch_state(self, stack_name, resources):
-        client = connect_to().cloudformation
-        child_stack_name = self.props["StackName"]
-        result = client.describe_stacks(StackName=child_stack_name)
-        result = (result.get("Stacks") or [None])[0]
-        return result
-
-    def get_cfn_attribute(self, attribute_name: str):
-        if attribute_name.startswith("Outputs."):
-            parts = attribute_name.split(".")
-            if len(parts) > 2:
-                raise Exception(
-                    f"Too many parts for stack output reference found: {attribute_name=}"
-                )
-            output_key = parts[1]
-            if "Outputs" not in self.props:
-                return None
-            candidates = [
-                o["OutputValue"] for o in self.props["Outputs"] if o["OutputKey"] == output_key
-            ]
-            if len(candidates) == 1:
-                return candidates[0]
-            else:
-                raise Exception(f"Too many output values found for key {output_key=}")
-
-        return super(CloudFormationStack, self).get_cfn_attribute(attribute_name)
+    # def fetch_state(self, stack_name, resources):
+    #     client = connect_to().cloudformation
+    #     child_stack_name = self.props["StackName"]
+    #     result = client.describe_stacks(StackName=child_stack_name)
+    #     result = (result.get("Stacks") or [None])[0]
+    #     return result
```

Comment:
commented out for now, I'll probably churn through the models and check where we can delete fetch_state implementations after the two PRs are merged :smile: